test/parallel/test-http-response-statuscode.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+const MAX_REQUESTS = 12;
+var reqNum = 0;
+
+const server = http.Server(common.mustCall(function(req, res) {
+  switch (reqNum) {
+    case 0:
+      assert.throws(common.mustCall(() => {
+        res.writeHead(-1);
+      }, /invalid status code/i));
```

Comment:
The closing parenthesis for `common.mustCall()` is misplaced here and on the other lines. It should read:

    }), /invalid status code/i);

With the current code, the RegExp is passed to `mustCall` as the `expected` paramater, and then silently replaced with `1` because it's non-numeric. Then the `assert.throws` call doesn't validate the error message and passes the test as long as *any* error is thrown.

I'll fix this along with my patch for https://github.com/nodejs/node/issues/9027 unless you'd prefer to see it in a separate PR.